src/App.tsx

```diff
@@ -15,48 +17,123 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Idle = 'idle',
+  Alphabet = 'alphabet',
+  Length = 'length',
+}
+
+interface Filter {
+  active: SortType;
+  isReversed: boolean;
+}
+
+const initialFilter: Filter = {
+  active: SortType.Idle,
+  isReversed: false,
+};
+
+const getGoodsForRender = ({ active, isReversed }: Filter): string[] => {
```

Comment:
You will always render the full list, so the name is not very appropriate
```suggestion
const getSortedGoods = ({ sortType, isReversed }: Filter): string[] => {
```